apps/enc.c

```diff
@@ -243,6 +246,11 @@ int enc_main(int argc, char **argv)
                 goto opthelp;
             cipher = c;
             break;
+        case OPT_ITER:
```

Comment:
I'm wondering whether we should have a "-pbkdf2" option that uses a default number of iterations as well?